### PR TITLE
Bridge: add bridge header classes

### PR DESCRIPTION
**Pull request by muleisheng**

### CODE REVIEW

**Comment by xiaoxq** [comment]
Please rebase your branch before submitting new code.

**Comment by jinghaomiao** [comment]
@muleisheng do a rebase before submitting the pull request.

File: modules/bridge/common/bridge_header_item.h [hunk 1]
```diff
@@ -0,0 +1,169 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
```
Comment (jinghaomiao) [inline modules/bridge/common/bridge_header_item.h]: 2019

**Comment by muleisheng** [comment]
> @muleisheng do a rebase before submitting the pull request.

Ok, I will be careful with pushing code

**Comment by muleisheng** [comment]
> Please rebase your branch before submitting new code.

OK, I will use rebase before pushing code.

**Pull request closed without merging**